imagevector/images.yaml

```diff
@@ -476,7 +476,7 @@ images:
     - type: 'githubTeam'
       teamname: 'gardener/logging-maintainers'
 - name: fluent-bit
-  sourceRepository: github.com/fluent/fluent-operator
+  sourceRepository: github.com/fluent/fluent-bit
```

Comment:
I am not sure this is correct. See https://github.com/gardener/ci-infra/blob/112affb951490ee7ecd4254ea4244291e003678c/config/images/images.yaml#L39.
I recall Kristian Z. when adding support for fluent-operator had to use this fluent-operator specific fluent-bit image.

@nickytd , correct me if I am wrong.

Reply:
We seem to use exactly https://github.com/fluent/fluent-operator/blob/master/cmd/fluent-watcher/fluentbit/Dockerfile:
https://github.com/gardener/gardener/blob/5f3113bd1667421f1d325dc1600dc5b1984e1a3c/pkg/component/observability/logging/fluentbit/fluentbit.go#L369-L370

Reply:
@ialidzhikov  That is correct. We are using the operator specific fluent-bit (which has the fluent-bit-watcher). Fluent-bit versions 2 does yet support config reload.

Reply:
The fluent-operator specific build is using the default fluent-bit image and adds the file-watcher ([ref](https://github.com/fluent/fluent-operator/blob/59a231611d9b2d86b24f90902d459b2a7abde679/cmd/fluent-watcher/fluentbit/Dockerfile#L9)). The main code base and all of the versioning information come from https://github.com/fluent/fluent-bit.
I don't want to change the container image we are using, this will still be the kubesphere built. I just want to change the github source repository.
If we keep https://github.com/fluent/fluent-operator as source repository for fluent-operator and fluent-bit, `renovate` won't be able to create update PRs. Their versions do not match.

Reply:
I don't recommend to have PRs for fluent-bit operator. There are a lot of moving bits there and it is not a simple dependency. Just an example I found and fixed an issue in the upstream project https://github.com/fluent/fluent-operator/pull/1160. So we skipped a release of the upstream. Usually I take a good time in researching in what is coming from upstream before deciding whether we shall move our stack or not. The upstream is not that mature compared to k8s for example. IMHO the PRs for the operator shall be a result of a deliberate effort only after we are sure that updates are ok.

Reply:
`renovate` could create PRs for patch releases only as we do it for istio and kubernetes dependencies.
In the current state fluent-bit is a patch release behind and fluent-operator images and go modules are not in sync.
WDYT?

Reply:
> I don't recommend to have PRs for fluent-bit operator.

We could still have them but we can continue to pay the attention we pay while reviewing and merging them.

---

In another world, I would leave the source repository as `github.com/fluent/fluent-operator` as it is where the Dockerfile and the watcher plugin live. But if using github.com/fluent/fluent-bit as source repository helps renovate somehow, then fine.

Have in mind also that not every container image tag in the `fluent/fluent-bit` repository is available under the `kubesphere/fluent-bit` repository.